eras/babbage/test-suite/test/Test/Cardano/Ledger/Babbage/GoldenTranslation.hs

```diff
@@ -0,0 +1,46 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE BlockArguments #-}
+{-# LANGUAGE ConstraintKinds #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+
+-- | Deserializes `TranslationInstance`s from golden/translations.cbor file.
+--
+-- Each instance represents arguments passed to `ExtendedUTxO.txInfo` along with the produced result.
+-- This test checks that calling `alonzoTxInfo` with the arguments from this file, produces the same result as in the flie.
+--
+-- To regenerate the golden file (for example, if the logic in the translation changes),
+-- run the following command from the root of the repository:
+-- cabal run cardano-ledger-<era>-test:gen-golden"
+module Test.Cardano.Ledger.Babbage.GoldenTranslation (
+  tests,
+)
+where
+
+import Test.Cardano.Ledger.Babbage.Serialisation.Generators ()
+
+import Cardano.Ledger.Babbage (Babbage)
+import Paths_cardano_ledger_babbage_test (getDataFileName)
+import Test.Cardano.Ledger.Alonzo.Translation.Golden (TxInfoResultComparison (..), compareGoldenTxInfoResults)
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.HUnit (Assertion, assertEqual, testCase)
+
+tests :: TestTree
+tests =
+  testGroup
+    "Golden translation tests"
+    [testCase "golden/translations.cbor" $ check "golden/translations.cbor"]
+
+check :: String -> Assertion
+check file = do
+  comps <- compareGoldenTxInfoResults @Babbage (getDataFileName file)
+  mapM_
+    ( \(TxInfoResultComparison expected actual err) ->
+        assertEqual err expected actual
+    )
+    comps
```

Comment:
That's the same function as in `Test.Cardano.Ledger.Alonzo.GoldenTranslation.goldenAssertion`

We should be able to parametrize it on `era`, put into a `Test.Cardano.Ledger.Alonzo.Translation.Golden` and reuse it here, right?